main.go

```diff
@@ -196,19 +164,21 @@ func main() {
 	fmt.Println()
 	log.Printf("System installed cocoapods version:")
 
-	podVersionCmdSlice := []string{"pod", "--version"}
+	if isCocoapodsInPath() {
+		podVersionCmdSlice := []string{"pod", "--version"}
 
-	log.Donef("$ %s", command.PrintableCommandArgs(false, podVersionCmdSlice))
+		log.Donef("$ %s", command.PrintableCommandArgs(false, podVersionCmdSlice))
 
-	cmd, err := rubycommand.New("pod", "--version")
-	if err != nil {
-		failf("Failed to create command model, error: %s", err)
-	}
+		cmd, err := rubycommand.New("pod", "--version")
```

Comment:
`log.Donef("$ %s", cmd.PrintableCommandArgs(false))` this way you can remove `podVersionCmdSlice := []string{"pod", "--version"}`